ykman-gui/qml/main.qml

```diff
@@ -121,7 +121,12 @@ ApplicationWindow {
         interval: 500
         repeat: true
         running: true
-        onTriggered: yubiKey.refresh()
+        onTriggered: yubiKey.refresh(function (resp) {
+            if (!resp.success) {
+                views.snackbarErrorMessage(resp)
+                views.home()
```

Comment:
I don't think this `views.home()` necessary, is it? Most of the time it'll just make the home view "flash" in place, and it could also disrupt some user flows in some situations. For example: Initiate FIDO reset, unplug YubiKey, plug in and quickly unplug again, then plug in properly. The refresh fails and dumps you back on the home screen instead of continuing with the reset.

Reply:
You're right!